dev/tracing_tests/test/image_cache_tracing_test.dart

```diff
@@ -99,7 +99,7 @@ bool _mapsEqual(Map<String, dynamic> expectedArgs, Map<String, dynamic> args) {
   return true;
 }
 
-// TODO(dnfield): we can drop this in favor of vm_service when https://github.com/dart-lang/webdev/issues/899 is resolved.
+// Change what now?
```

Comment:
@dnfield here too

Reply:
I believe we're now on package:vmservice >= 3.0.0.  We can drop all the custom logic in here to fetch the timeline, and instead use `package:vmservice`.